web/modules/custom/customnewblock/customnewblock.install

```diff
@@ -0,0 +1,78 @@
+<?php
+
+use Drupal\block\Entity\Block;
+use Drupal\Core\Config\FileStorage;
+/**
```

Comment:
@anitha-innoraft - This doesn't follow coding standards. There should be a gap of one line after use statement.

Reply:
Fixed